awswrangler/_distributed.py

```diff
@@ -0,0 +1,94 @@
+"""Distributed Module (PRIVATE)."""
+
+import multiprocessing
+import os
+import sys
+import warnings
+from typing import TYPE_CHECKING, Any, Callable, Optional
+
+import psutil
+
+from awswrangler._config import apply_configs, config
+
+if config.distributed or TYPE_CHECKING:
+    import ray
+
+
+def _ray_remote(function: Callable[..., Any]) -> Any:
```

Comment:
Since both `_ray_remote` and `_initialize_ray` are used in other modules shall we make them public (no underscore)?

Reply:
They won't be exposed to the user so not sure why we should make them public. There are plenty of example of private methods in the codebase that are reused yet not exposed and left private